diff -aurN lightdm-gtk-greeter-1.6.1.orig/data/lightdm-gtk-greeter.conf lightdm-gtk-greeter-1.6.1/data/lightdm-gtk-greeter.conf
--- lightdm-gtk-greeter-1.6.1.orig/data/lightdm-gtk-greeter.conf	2013-09-26 11:50:59.798142683 +0200
+++ lightdm-gtk-greeter-1.6.1/data/lightdm-gtk-greeter.conf	2013-09-26 11:51:44.925872340 +0200
@@ -14,7 +14,7 @@
 # keyboard = command to launch on-screen keyboard
 #
 [greeter]
-#background=
+background=/usr/share/pixmaps/lightdm_bg.png
 #theme-name=
 #icon-theme-name=
 #font-name=
@@ -22,8 +22,8 @@
 #xft-dpi=
 #xft-hintstyle=
 #xft-rgba=
-#show-language-selector=
-#show-indicators=
-#show-clock=
+show-language-selector=true
+show-indicators=
+show-clock=true
 #clock-format=
 #keyboard=
