diff -aurN lightdm-gtk-greeter-1.6.0.orig/data/lightdm-gtk-greeter.conf lightdm-gtk-greeter-1.6.0/data/lightdm-gtk-greeter.conf
--- lightdm-gtk-greeter-1.6.0.orig/data/lightdm-gtk-greeter.conf	2013-08-09 23:33:58.013423566 +0200
+++ lightdm-gtk-greeter-1.6.0/data/lightdm-gtk-greeter.conf	2013-08-09 23:34:47.486274407 +0200
@@ -14,17 +14,17 @@
 # keyboard = command to launch on-screen keyboard
 #
 [greeter]
-#background=
-#theme-name=
-#icon-theme-name=
-#cursor-theme-name=
-#font-name=
-#xft-antialias=
-#xft-dpi=
-#xft-hintstyle=
-#xft-rgba=
+background=/usr/share/pixmaps/lightdm_bg.png
+theme-name=Shiki-Brave
+icon-theme-name=gnome-brave
+cursor-theme-name=OpenZone_Black
+font-name=Droid Sans 9
+xft-antialias=true
+xft-dpi=96
+xft-hintstyle=slight
+xft-rgba=rgb
 #show-language-selector=
 #show-indicators=
-#show-clock=
+show-clock=true
 #clock-format=
 #keyboard=
